packages/block-library/src/button/style.scss

```diff
@@ -1,4 +1,4 @@
-$blocks-button__height: 56px;
+$blocks-button__height: 3.1em;
```

Comment:
There are several numbers throughout this PR that aren't perfect. In this case, the correct value would be 3.1111111... but the question is how many decimal places should we keep? I've noticed in some cases where the calculation doesn't end in repeating digits, you went for 3 decimal places. So why only 1 here?

Reply:
The initial value could be revisited, to make for cleaner em conversions.

Reply:
well, 3.1 is approximately the same as 3.111... 
If it was 3.1555 then I'd definitely add an extra digit, but in this case it didn't make much sense.
I agree that the actual values can be rounded to make more sense.
This PR is a direct conversion from px to em without changing a lot of things, but if we have the OK from design to tweak the values I believe we can make the values a bit more sensible and meaningful :+1: